src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcTokenHandlerFactory.php

```diff
@@ -67,7 +68,12 @@ public function addConfiguration(NodeBuilder $node): void
                     ->end()
                     ->scalarNode('audience')
```

Comment:
this should also accept a list of strings as tokens can target multiple recipients. See https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3

Reply:
> Each principal intended to process the JWT MUST identify itself with a value in the audience claim.

As I understand, the `aud` claim in the token can be a string (if it's valid for a single audience) or an array (if it's valid for multiple audiences). But each audience must be identified by a single value, which should match the claim (if string) or included in it (if array).